src/components/Widgets/ControlWidget/PlayBackPage/index.js

```diff
@@ -28,15 +24,7 @@ const ControlWidget = ({
         {candidate}
       </span>
     </div>
-    <div>
-      <InputGroup compact style={{ width: 'auto', display: 'inline-block' }}>
-        <RecordSelector
-          onChange={onChangeRecord}
-          recordIndex={recordIndex}
-          list={recordList}
-        />
-      </InputGroup>
-    </div>
+    <div></div>
```

Comment:
Did you remove this accidentally?
Looks like it's for playback functionalitity.